backend/python/app/graphql/queries/story_query.py

```diff
@@ -15,3 +15,7 @@ def resolve_stories_available_for_translation(root, info, language, level):
 
 def resolve_story_translations_by_user(root, info, user_id, translator):
     return services["story"].get_story_translations(user_id, translator)
+
+
+def resolve_story_available_for_review_by_user(root, info, language, level):
```

Comment:
nit: rename to `resolve_story_translations_available_for_review`
dont forget to change the name in all uses of the function